openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -915,4 +923,15 @@ private OpenApiStaticFile getDefaultStaticFile() {
             return null;
         }
     }
+
+    private static <T> T access(Semaphore guard, Supplier<T> operation) {
+        try {
+            guard.acquire();
+            T result = operation.get();
+            guard.release();
```

Comment:
This will not happen if `operation.get()` fails.

Reply:
Fixed. Using lock.